### PR TITLE
Add test plan key configurability

### PR DESCRIPTION
**Pull request by csvtuda**
This PR adds the option to configure an Xray test plan key, to which test execution issues will be attached on results upload.

### CODE REVIEW

File: qtaf-xray-plugin/src/test/java/de/qytera/qtaf/xray/config/XrayConfigHelperTest.java [hunk 1]
```diff
@@ -16,4 +23,12 @@ public void testService() {
         configMap.setString(XrayConfigHelper.XRAY_SERVICE_SELECTOR, "cloud");
         Assert.assertEquals(XrayConfigHelper.getXrayService(), "cloud");
     }
+
```
Comment (qytera-development) [inline qtaf-xray-plugin/src/test/java/de/qytera/qtaf/xray/config/XrayConfigHelperTest.java:26]: Please add a comment to the method

**Review by qytera-development — COMMENTED** [review]
Please add comments to the methods

File: qtaf-xray-plugin/src/test/java/de/qytera/qtaf/xray/config/XrayConfigHelperTest.java (hunk 1 above)
Comment (csvtuda) [reply qtaf-xray-plugin/src/test/java/de/qytera/qtaf/xray/config/XrayConfigHelperTest.java:26]: Fixed in [6378a3c](https://github.com/Qytera-Gmbh/QTAF/pull/85/commits/6378a3cac4571064d8657cb29ba2ecfa3e0589e2).

File: qtaf-xray-plugin/src/test/java/de/qytera/qtaf/xray/config/XrayConfigHelperTest.java [hunk 2]
```diff
@@ -3,9 +3,16 @@
 import de.qytera.qtaf.core.QtafFactory;
 import de.qytera.qtaf.core.config.entity.ConfigMap;
 import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 public class XrayConfigHelperTest {
+
```
[the remark this answers is not in the document]
Comment (csvtuda) [reply qtaf-xray-plugin/src/test/java/de/qytera/qtaf/xray/config/XrayConfigHelperTest.java:10]: Fixed in [6378a3c](https://github.com/Qytera-Gmbh/QTAF/pull/85/commits/6378a3cac4571064d8657cb29ba2ecfa3e0589e2).

File: qtaf-xray-plugin/src/test/java/de/qytera/qtaf/xray/builder/XrayJsonImportBuilderTest.java [hunk 3]
```diff
@@ -393,4 +393,14 @@ public void testMultipleRunStepMerge() throws XrayJsonImportBuilder.NoXrayTestEx
         Assert.assertEquals(dto.getTests().get(0).getIterations().get(0).getSteps().get(0).getAllEvidence().size(), 3);
     }
```
[the remark this answers is not in the document]
Comment (csvtuda) [reply qtaf-xray-plugin/src/test/java/de/qytera/qtaf/xray/builder/XrayJsonImportBuilderTest.java:399]: Fixed in [6378a3c](https://github.com/Qytera-Gmbh/QTAF/pull/85/commits/6378a3cac4571064d8657cb29ba2ecfa3e0589e2).

**Review by mzdih — APPROVED** [review]
PR nicht reviewed, nur approved

**Pull request merged**